src/main/java/hudson/plugins/git/UserMergeOptions.java

```diff
@@ -36,22 +37,25 @@
      * @param mergeStrategy merge strategy to be used
      */
     @Deprecated
-    public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStrategy) {
-        this(mergeRemote, mergeTarget, mergeStrategy, MergeCommand.GitPluginFastForwardMode.FF);
+    public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStrategy, Integer timeout) {
+        this(mergeRemote, mergeTarget, mergeStrategy, MergeCommand.GitPluginFastForwardMode.FF, timeout);
```

Comment:
You shouldn't change the signature of a deprecated function, it should just specify a sane default